library/src/main/res/values/strings.xml

```diff
@@ -30,13 +30,16 @@
     <string name="location_disabled_location_permissions_not_granted">Sorry but we could not get your location since the app does not have permissions to access your Location</string>
     <string name="current_location_btn_visibility">current_location_btn_visibility</string>
     <string name="mapbox_warmGps">mapbox_warmGps</string>
-    <string name="location_permission">Location Permission</string>
-    <string name="location_permission_reason">We need the location permission to focus on your location in the map</string>
+    <string name="kujaku_permission">Location &amp; Storage Permissions</string>
+    <string name="kujaku_permission_reason">We need the location &amp; storage permissions to run the kujaku MapView</string>
```

Comment:
Add a user friendly reason for this. The user's do not know what the Kujaku MapView is. Change this to

`We need the location permission to focus on your location in the map. The storage permission is required to store you route during route-tracking`

Reply:
ok make sense